app/settings/data-export/hdx-export.controller.js

```diff
@@ -108,23 +108,70 @@ function (
     }
 
     function selectTag(attribute) {
-        attribute.selected = true;
-        if (attribute.selectedTag && attribute.selectedTag.tag_name) {
-            attribute.pretty = '#' + attribute.selectedTag.tag_name;
-        } else {
-            attribute.pretty = '';
+        attribute.selected = [attribute.id];
+        attribute.hxl_label = createHxlLabel(attribute);
+        if (!attribute.selectedTag || !attribute.selectedTag.tag_name) {
+            addAnother(attribute);
         }
-        attribute.nbAttributes = 1;
-        attribute.selectedHxlAttributes = {};
+
+        attribute.selectedHxlAttributes = [];
     }
 
     function selectHxlAttribute(attribute) {
-        attribute.pretty = '#' + attribute.selectedTag.tag_name;
+        if (!needsMatchedAttribute(attribute)) {
+            attribute.hxl_label = createHxlLabel(attribute);
+        } else {
+            attribute = addGeoMatchedAttribute(attribute.selectedHxlAttributes[attribute.selectedHxlAttributes.length - 1], attribute);
```

Comment:
uh wait I think I should just grab this attribute.selectedHxlAttributes[attribute in the addGeoMatchedAttribute function